tests/test_postgresql.py

```diff
@@ -593,6 +593,7 @@ def time_in_state(*args):
             self.p._state = 'starting'
             self.assertIsNone(self.p.wait_for_startup())
 
+    @patch.object(Postgresql, '_get_synchronous_commit_param', Mock(return_value='remote_apply'))
```

Comment:
Lets also change this test to:
```python
    def test_pick_sync_standby(self):                                                                                                   
        cluster = Cluster(True, None, self.leader, 0, [self.me, self.other, self.leadermem], None,                                      
                          SyncState(0, self.me.name, self.leadermem.name), None)                                                        
        mock_cursor = Mock()                                                                                                            
        mock_cursor.fetchone.return_value = ('remote_apply',)                                                                           
                                                                                                                                        
        with patch.object(Postgresql, "query", side_effect=[                                                                            
                    mock_cursor,                                                                                                        
                    [(self.leadermem.name, 'streaming', 'sync'),                                                                        
                     (self.me.name, 'streaming', 'async'),                                                                              
                     (self.other.name, 'streaming', 'async')]                                                                           
                ]):                                                                                                                     
            self.assertEqual(self.p.pick_synchronous_standby(cluster), ([self.leadermem.name], [self.leadermem.name]))                  
                                                                                                                                        
        with patch.object(Postgresql, "query", side_effect=[                                                                            
                    mock_cursor,                                                                                                        
                    [(self.leadermem.name, 'streaming', 'potential'),                                                                   
                     (self.me.name, 'streaming', 'async'),                                                                              
                     (self.other.name, 'streaming', 'async')]                                                                           
                ]):                                                                                                                     
            self.assertEqual(self.p.pick_synchronous_standby(cluster), ([self.leadermem.name], []))                                     
                                                                                                                                        
        with patch.object(Postgresql, "query", side_effect=[                                                                            
                    mock_cursor,                                                                                                        
                    [(self.me.name, 'streaming', 'async'),                                                                              
                     (self.other.name, 'streaming', 'async')]                                                                           
                ]):                                                                                                                     
            self.assertEqual(self.p.pick_synchronous_standby(cluster), ([self.me.name], []))                                            
                                                                                                                                        
        with patch.object(Postgresql, "query", side_effect=[                                                                            
                    mock_cursor,                                                                                                        
                    [('missing', 'streaming', 'sync'),                                                                                  
                     (self.me.name, 'streaming', 'async'),                                                                              
                     (self.other.name, 'streaming', 'async')]                                                                           
                ]):                                                                                                                     
            self.assertEqual(self.p.pick_synchronous_standby(cluster), ([self.me.name], []))                                            
                                                                                                                                        
        with patch.object(Postgresql, "query", side_effect=[mock_cursor, []]):                                                          
            self.p._major_version = 90400                                                                                               
            self.assertEqual(self.p.pick_synchronous_standby(cluster), ([], []))                                                        
```

I just want that `pick_synchronous_standby()` also calls `_get_synchronous_commit_param()`method.

Reply:
fixed